core/src/main/scala/quasar/sql/fixpoint.scala

```diff
@@ -62,8 +62,14 @@ object ArrayLiteral {
     ArrayLiteralF.unapply(obj.unFix)
 }
 
+/**
+ * Represents the wildcard in a select projection
+ * For instance:
+ *  "select foo.* from example" => ...(Splice(Some(Ident("foo"))))...
+ *  "select * from example"     => ...(Splice(None))...
+ */
```

Comment:
Ok, will do, always did doc comments like this, but I see that is not the standard formatting.